src/trackers/customBackend.js

```diff
@@ -0,0 +1,38 @@
+var util = require("../util");
+var enumerable = require("../enumerable");
+
+module.exports = function (category, action, label, value) {
+  var logLabel = "Bighorn.track customBackend";
+  var backendUrl  = Bighorn.customUrl;
+
+  console.log("PRE", logLabel, backendUrl);
+
+  try {
+    if (!util.isValidString(backendUrl)) { return; }
+
+    var data = {
+      category: category,
+      action: action,
+      label: label,
+      value: value
+    };
+
+    var properties = {};
+
+    enumerable.each(data, function (key, value) {
```

Comment:
This code for flattening a single dimension of the data object is copied from the ahoy.js backend. I wasn't sure if it made sense to pull it out into it's own utility function.